__tests__/testCases/withPensionInput.ts

```diff
@@ -1,9 +1,10 @@
-import { TaxFormUserInput } from '../../src/types/TaxFormUserInput';
+import {TaxFormScenario} from "../types/TaxFormScenario";
 
-export const withPensionInput: TaxFormUserInput = {
+export const withPensionInput: TaxFormScenario = {
   priloha3_r11_socialne: '1000',
   priloha3_r13_zdravotne: '1000',
   t1r10_prijmy: '25000',
+  t1r10_prijmy_formatted: '25 000,00 EUR',
```

Comment:
Hmm preco by toto malo byt v data modeli? Nebolo by lepsie pouzit formatovaciu funkciu vsade kde to cislo chceme zobrazit v tomto formate? Inac miesame zobrazovaciu logiku s datami.

Reply:
Nechcel som sa spoliehat na formatovaciu funckiu. Ale na druhej strane ta funkcia ma svoj unit test, takze by to mohlo byt ok. Zmenim 👍

Reply:
Preco by sa na nu nemalo dat spolahnut?

Reply:
Pretoze ak bude bug v tej funkcii, tak tento test vzdy prejde (lebo bude pouzivat tu bugnutu funkciu). Ale kedze ma vlastny unit test, tak je to asi ok.

Reply:
Aha, uz tomu rozumiem. Hej no, to je pravda.